collectors/osd.go

```diff
@@ -533,6 +611,19 @@ func (o *OSDCollector) cephOSDPerfCommand() []byte {
 	return cmd
 }
 
+func (o *OSDCollector) cephPGDumpCommand() []byte {
+	cmd, err := json.Marshal(map[string]interface{}{
+		"prefix": "pg dump",
```

Comment:
Can we do `pg dump pgs_brief` here? Might be useful for larger cluster as it will surface lesser data and will complete relatively quickly, as this will be run periodically.